catalog/spatial/csw/spatial-csw-source/pom.xml

```diff
@@ -124,7 +129,7 @@
         <dependency>
             <groupId>xmlunit</groupId>
             <artifactId>xmlunit</artifactId>
-            <version>1.6</version>
+            <version>1.4</version>
```

Comment:
I was told that 1.4 was the version we are using everywhere else so I changed it to be consistent